hopsworks-web/yo/app/scripts/controllers/experimentCtrl.js

```diff
@@ -232,6 +232,10 @@ angular.module('hopsWorksApp')
                                 angular.forEach(success.data.items, function (experiment, key) {
                                     if(typeof self.experiments[i] === 'undefined') {
```

Comment:
@robzor92 just to be on the safe side, we add more checks like these https://github.com/logicalclocks/hopsworks/blob/master/hopsworks-web/yo/app/scripts/controllers/servingCtrl.js#L202

Reply:
added those checks too